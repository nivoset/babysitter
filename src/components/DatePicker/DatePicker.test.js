import React from 'react';
import { create } from 'react-test-renderer';
import { MuiPickersUtilsProvider } from "@material-ui/pickers";
import {DatePicker} from '.';
import DateFnsUtils from '@date-io/date-fns';

test('renders learn react link', () => {
  console.warn(create)
  const component = create((
    <MuiPickersUtilsProvider utils={DateFnsUtils}>
      <DatePicker time={new Date(0)} updateFunction={() => {}} />
    </MuiPickersUtilsProvider>));
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
