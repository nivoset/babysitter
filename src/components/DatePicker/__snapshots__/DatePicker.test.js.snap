// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
  format="hh:mm a"
  onClick={[Function]}
  onKeyDown={[Function]}
>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      readOnly={true}
      required={false}
      type="text"
      value="07:00 PM"
    />
  </div>
  
</div>
`;
